src/features/reports/meeting_attendance/export_S88/useExportS88.tsx

```
@@ -0,0 +1,389 @@
+
+    let count = 0;
+
+    for (let i = 1; i <= 5; i++) {
+      const weekData = attendance[`week_${i}`] as WeeklyAttendance;
+      const meetingData = weekData.midweek;
+
+      const total = meetingData.reduce((acc, current) => {
+        let value = acc;
+
+        if (current?.present) {
+          value += value + current.present;
```

Biome (1.9.4)

[error] Variable appears on both sides of an assignment operation.

This assignment might be the result of a wrong refactoring.
Unsafe fix: Use value += current.present instead.


(lint/suspicious/noMisrefactoredShorthandAssign)